rdrf/rdrf/services/io/defs/importer.py

```diff
@@ -540,55 +540,30 @@ def _create_forms(self, registry, imported_forms):
             imported_forms.add(f.name)
 
     def _create_all_remaining_objects(self, r):
-        if "demographic_fields" in self.data:
-            self._create_demographic_fields(self.data["demographic_fields"])
-            logger.info("demographic field definitions OK ")
-        else:
-            logger.info("no demographic_fields to import")
-
-        if "complete_fields" in self.data:
-            self._create_complete_form_fields(r, self.data["complete_fields"])
-            logger.info("complete field definitions OK ")
-        else:
-            logger.info("no complete field definitions to import")
-
-        if "reports" in self.data:
-            self._create_reports(self.data["reports"])
-            logger.info("complete reports OK ")
-        else:
-            logger.info("no reports to import")
-
-        if "cde_policies" in self.data:
-            self._create_cde_policies(r)
-            logger.info("imported cde policies OK")
-        else:
-            logger.info("no cde policies to import")
-
-        if "context_form_groups" in self.data:
-            self._create_context_form_groups(r)
-            logger.info("imported context form groups OK")
-        else:
-            logger.info("no context form groups to import")
-
-        if "email_notifications" in self.data:
-            self._create_email_notifications(r)
-            logger.info("imported email notifications OK")
-
-        if "consent_rules" in self.data:
-            self._create_consent_rules(r)
-            logger.info("imported consent rules OK")
-
-        if "surveys" in self.data:
-            self._create_surveys(r)
-            logger.info("imported surveys OK")
-
-        if "reviews" in self.data:
-            self._create_reviews(r)
-            logger.info("imported reviews OK")
-
-        if "custom_actions" in self.data:
-            self._create_custom_actions(r)
-            logger.info("imported custom actions OK")
+        remaining_objects = [
+            "demographic_fields",
+            "complete_fields",
+            "reports",
+            "cde_policies",
+            "context_form_groups",
+            "email_notifications",
+            "consent_rules",
+            "surveys",
```

Comment:
yeah this seems ok - or you could use some other data structure to store a bit more config for the exceptional cases, a dict maybe